source/features/hide-watch-and-fork-count.tsx

```diff
@@ -0,0 +1,13 @@
+import features from '../libs/features';
+import select from 'select-dom';
+
+function init() {
+	for (const btn of select.all('.social-count[href$="/network/members"], .social-count[href$="/watchers"]')) {
```

Comment:
Like for https://github.com/sindresorhus/refined-github/blob/master/source/features/hide-useless-newsfeed-events.tsx, the class can just be added to `body` so we don't have to re-select the items in Javascript.